orc8r/gateway/python/magma/magmad/sync_rpc_client.py

```diff
@@ -129,20 +129,24 @@ def forward_requests(self,
         end of the iterator sync_rpc_requests, or encounters an error)
 
         """
-        try:
-            while True:
-                logging.debug("[SyncRPC] Waiting for requests")
+        logging.info("[SyncRPC] Waiting for requests")
+        while True:
+            try:
```

Comment:
Did you flip the order of the try and while just as a sanitation thing? Since you're re-raising in the except still, this loop will still stop if there's anexception